docs/proposals/PROPOSAL_A.md

```diff
@@ -0,0 +1,289 @@
+# Proposal A
+
+This proposal defines a new artifact manifest and corresponding referrers extension API for registries to support reference types.
+
+The modifications include:
+
+- [New artifact manifest](#artifact-manifest)
+- [Descriptor (OPTIONAL) additional property](#descriptor-properties)
+- [Distribution - Referrers API](#registry-http-api)
+
+## Artifact Manifest
+
+Provides an optional `subject` reference to another manifest (runtime image), an `artifactType` to differentiate types of artifacts (signatures, sboms, security scan results) and renames `layers` to optional `blobs`.
+
+## Links
+
+| Description                                 | Link                        |
+| ------------------------------------------- | --------------------------- |
+| GitHub issue where this was first proposed:  | [OCI Artifact Manifest - with weak reference support #27](https://github.com/opencontainers/artifacts/pull/27)<br>[OCI Artifact Manifest, Phase 1-Reference Types #29](https://github.com/opencontainers/artifacts/pull/29) |
+| Current ORAS Specifications:   |  [Manifest](https://github.com/oras-project/artifacts-spec/blob/main/artifact-manifest.md) and [Referrers API](https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md) | 
+| Implementations | Registry: [ORAS Fork of CNCF Distribution (WIP)](https://github.com/oras-project/distribution/tree/ext_oras)<br>Registry: [ZOT Project (merged)](https://github.com/project-zot/zot/issues/264)<br>Registry client: [oras-project/oras/tree/artifacts](https://github.com/oras-project/oras/tree/artifacts) |
+
+### Modifications 
+
+The Artifact manifest is similar to the [OCI image manifest][oci-image-manifest-spec], but removes constraints defined on the image-manifest such as a required `config` object and required & ordinal `layers`.
+It adds a `subject` property supporting a graph of independently linked artifacts.
+The addition of a new manifest does not change, nor impact the `image.manifest`.
+It provides a means to define a wide range of artifacts, including a chain of related artifacts enabling SBoMs, signatures and metadata that can be related to an `image.manifest`, `image.index` or another `artifact.manifest`.
+By defining a new manifest, registries and clients opt-into new capabilities, without breaking existing registry and client behavior or setting expectations for scenarios to function when the client and/or registry may not yet implement new capabilities
```

Comment:
nothing here

Reply:
I don't follow - can you provide a little more detail here @jdolitsky so that we can action?

Reply:
Sorry, meant that this would preferably nest underneath one of the subsections for "Modifications" (Schema or Registry API), if possible

Reply:
Is this about matching the schema of the template? 

Is the information not helpful? This proposal adds a new manifest, which is assumed complex. This is why we've added this text to explain.

Reply:
I am barely reading the actually contents of the proposal at this point. My review comments are entirely cosmetic as it relates to conforming to the template.